third_party/blink/renderer/platform/media/web_media_player_impl.cc

```diff
@@ -2805,6 +2805,9 @@ std::unique_ptr<media::Renderer> WebMediaPlayerImpl::CreateRenderer(
   // TODO(b/375278384): Select the StarboardRenderer properly instead of
   //                    hard coding.
 
+  // StarboardRenderer always uses full screen with overlay video mode.
```

Comment:
Is this always true? IOW if we are playing back windowed video using  
StarboardRenderer, it would not use overlays?

Reply:
Right now, yes, it will work using overlay video mode.

The case of secondary video is not supported yet, but all the videos for Cobalt.apk with StarboardRenderer use overlay video mode.

Reply:
Gotcha thanks!

Just to be super sure, if overlays are needed for protected  
content (big if, but I think true), would this mean protected  
content playback won't work in a windowed <video> ?

Reply:
As secondary video is not supported at the moment (it must use decode-to-texture on ATV), all the video (only one on Cobalt.apk) including protected content can use video overlay without issues, because cc just punches a video hole for SbPlayer, and as long as SbPlayer can play the video with the right bounds, it won't be the issue.

The open question here is for picture-in-picture mode, would StarboardRenderer work?